src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int width;
+    private String color = getColor();
+
+    public Rectangle(int height, int width, String color) {
+        this.height = height;
+        this.width = width;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (height * width);
```

Comment:
The method name 'areaCalculator' sounds like a verb. It would be better to name it 'calculateArea' to follow the Java naming conventions for methods.